src/recordlinker/linking/mpi_service.py

```diff
@@ -133,7 +134,7 @@ def bulk_insert_patients(
     pat_data = [
         {
             "person_id": person.id,
-            "_data": record.to_json(prune_empty=True),
+            "_data": json.loads(record.to_json(prune_empty=True)),
```

Comment:
@ericbuckley small bug I found in the `bulk_insert_patients` function. `_data` was being stored as a string instead of a `dict`. This caused a type error to be thrown whenever an incoming record was being compared to a patient since this line `self._record = pii.PIIRecord.model_construct(**(self._data or {}))` requires `self._data` to be a dict.

Reply:
fixed in a different PR and rebased